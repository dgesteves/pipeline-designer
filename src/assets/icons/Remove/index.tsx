import { TProps } from "../types";

export default function Remove(props: TProps): JSX.Element {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 32 32"
      width="1em"
      height="1em"
      {...props}
    >
      <path
        style={{
          fill: "currentColor",
        }}
        d="M12 12h2v12h-2zM18 12h2v12h-2z"
      />
      <path
        style={{
          fill: "currentColor",
        }}
        d="M4 6v2h2v20a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V8h2V6Zm4 22V8h16v20ZM12 2h8v2h-8z"
      />
      <path
        data-name="&lt;Transparent Rectangle&gt;"
        style={{
          fill: "none",
        }}
        d="M0 0h32v32H0z"
      />
    </svg>
  );
}
