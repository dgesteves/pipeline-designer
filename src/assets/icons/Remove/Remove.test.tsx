import { render } from "@testing-library/react";
import Remove from "./index";

describe("Remove icon", () => {
  it("should render correctly", () => {
    const { container } = render(<Remove />);
    expect(container).toBeInTheDocument();
  });

  it("should have the correct svg", () => {
    const { container } = render(<Remove />);
    expect(container).toMatchSnapshot();
  });
});
