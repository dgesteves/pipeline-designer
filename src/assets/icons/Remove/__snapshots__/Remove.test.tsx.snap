// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remove icon should have the correct svg 1`] = `
<div>
  <svg
    height="1em"
    viewBox="0 0 32 32"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M12 12h2v12h-2zM18 12h2v12h-2z"
      style="fill: currentColor;"
    />
    <path
      d="M4 6v2h2v20a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V8h2V6Zm4 22V8h16v20ZM12 2h8v2h-8z"
      style="fill: currentColor;"
    />
    <path
      d="M0 0h32v32H0z"
      data-name="<Transparent Rectangle>"
      style="fill: none;"
    />
  </svg>
</div>
`;
